### ISSUE TITLE
softmax activation on very last layer

### ISSUE THREAD

**Issue by gihyunkim** [4uiiurz1/pytorch-adacos#14]
is it not supposed to use softmax function on last layer? I am confused that someone use softmax on last layer and others don't. Does it gives different result whether I use softmax or not? or it just for convenience to evaluate. plz let me know

**Comment by smile-cpu** [comment]
使用交叉熵损失函数，交叉熵损失函数自带softmax()

**Comment by smile-cpu** [comment]
使用交叉熵损失函数，交叉熵损失函数自带softmax()
